docs/selection_example.py

```diff
@@ -25,7 +25,7 @@ def adaptive_introgression(seed):
     # One mutation type, which we'll use for the positively selected mutation.
     # Neutral mutations will be added by the SLiM engine as usual, after the
```

Comment:
I think we can retire this now, no? We have tutorials for all of these things, and most of these examples are outdaded.

Reply:
*retire this file

Reply:
agreed!

Reply:
I'll remove the DFE stuff but not the adaptive introgression stuff quite yet - that's not so obviously covered elsewhere.